packages/orchestration/src/facade.js

```diff
@@ -230,7 +231,18 @@ export const makeOrchestrationFacade = ({
         getBrandInfo: anyVal,
         asAmount: anyVal,
       };
-      return async (...args) => fn(orc, ctx, ...args);
+
+      const { asyncFlow } = asyncFlowTools;
+      /** @type {(...args: Args) => Promise<unknown>} */
+      // TODO The lexical references to `orc` and `ctx` below are a
+      // stopgap for a correct endowments mechanism. This stopgap passes
+      // them through directly, rather than through the membrane. As a result,
+      // interactions with these endowments will not yet be replayable.
+      // Even as a stopgap, this will bypass too much. But we gotta get
+      // started somewhere.
+      const guestFunc = async (...args) => fn(orc, ctx, ...args);
+      const hostFunc = asyncFlow(zone, durableName, guestFunc);
+      return hostFunc;
```

Comment:
Here is what I had in my draft:

```suggestion
      const subzone = zone.subZone(durableName);

      // TODO: create durable exo wrappers for ctx props
      const ctxKit = {}; // makeDurableCtxKit(ctx);

      const asyncFlowGuestFunc = async (flowSupportKit, ...args) => {
        // TODO: Add support for recreating a `this` exo class context
        const exoContext = undefined;
        // TODO: Get orc from `flowSupportKit` once it's made durable
        const orcArg = orc;
        // TODO: build ctx from a durable skeleton using `flowSupportKit`
        const ctxArg = ctx;
        return apply(fn, exoContext, [orcArg, ctxArg, ...args]);
      }

      const asyncFlowHostFn = asyncFlowFake(subzone, 'asyncFlow', asyncFlowGuestFunc);

      return harden({
        [durableName](...args) {
          // TODO: Build the needed durable objects
          const orcKit = {}; // internalMakeOrchestratorKit();

          // TODO: plumb the necessary ingredients to reconstruct a `this` exo context inside the guest

          return asyncFlowHostFn({ orc: orcKit.orchestrator, ctxKit }, ...args);
        }, 
      }[durableName]);
```

Reply:
See https://github.com/Agoric/agoric-sdk/pull/9566 , where this work continues.